### ISSUE TITLE
[pytest] prepare_testbed_ssh_keys did not catch RunAnsibleModuleFail exeption

### ISSUE THREAD

**Issue by msosyak** [Azure/sonic-mgmt#1669]
**Description**
The `prepare_testbed_ssh_keys` calls shell module with cmd `stat /root/.ssh/known_hosts` and except `RunAnsibleModuleFail`.
In case when cmd failed `RunAnsibleModuleFail` is raised here https://github.com/Azure/sonic-mgmt/blob/master/tests/common/devices.py#L63 but not caught by `expect RunAnsibleModuleFail` here https://github.com/Azure/sonic-mgmt/blob/master/tests/common/platform/ssh_utils.py#L21. 

**Steps to reproduce the issue:**
1.Run any test that used `prepare_testbed_ssh_keys`. (I run fast-reboot)
2.
3.

**Describe the results you received:**
```
___________________________________________________________________________________ test_fast_reboot ___________________________________________________________________________________

request = <FixtureRequest for <Function test_fast_reboot>>, get_advanced_reboot = <function get_advanced_reboot at 0x7f95f8c532a8>

    @pytest.mark.usefixtures('get_advanced_reboot')
    def test_fast_reboot(request, get_advanced_reboot):
        '''
        Fast reboot test case is run using advacned reboot test fixture
    
        @param request: Spytest commandline argument
        @param get_advanced_reboot: advanced reboot test fixture
        '''
        advancedReboot = get_advanced_reboot(rebootType='fast-reboot')
>       advancedReboot.runRebootTestcase()

platform/test_advanced_reboot.py:12: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
common/fixtures/advanced_reboot.py:377: in runRebootTestcase
    self.__setupTestbed()
common/fixtures/advanced_reboot.py:290: in __setupTestbed
    self.__prepareTestbedSshKeys()
common/fixtures/advanced_reboot.py:221: in __prepareTestbedSshKeys
    prepareTestbedSshKeys(self.duthost, self.ptfhost, self.rebootData['dut_username'])
common/platform/ssh_utils.py:21: in prepare_testbed_ssh_keys
    ptfhost.shell('stat /root/.ssh/known_hosts')
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tests.common.devices.PTFHost object at 0x7f95faa6ba10>, module_args = ('stat /root/.ssh/known_hosts',), complex_args = {}, module_ignore_errors = False, module_async = False
res = {'stderr_lines': [u"stat: cannot stat '/root/.ssh/known_hosts': No such file o...': True, u'stdin': None}}, 'stdout_lines': [], u'msg': u'non-zero return code'}

    def _run(self, *module_args, **complex_args):
        module_ignore_errors = complex_args.pop('module_ignore_errors', False)
        module_async = complex_args.pop('module_async', False)
    
        if module_async:
            def run_module(module_args, complex_args):
                return self.module(*module_args, **complex_args)[self.hostname]
            pool = ThreadPool()
            result = pool.apply_async(run_module, (module_args, complex_args))
            return pool, result
    
        res = self.module(*module_args, **complex_args)[self.hostname]
        if res.is_failed and not module_ignore_errors:
>           raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), res)
E           RunAnsibleModuleFail: run module shell failed
E           Ansible Results => changed=true 
E             cmd: stat /root/.ssh/known_hosts
E             delta: '0:00:00.113402'
E             end: '2020-05-13 08:53:51.242776'
E             msg: non-zero return code
E             rc: 1
E             start: '2020-05-13 08:53:51.129374'
E             stderr: 'stat: cannot stat ''/root/.ssh/known_hosts'': No such file or directory'
E             stderr_lines: <omitted>
E             stdout: ''
E             stdout_lines: <omitted>

common/devices.py:63: RunAnsibleModuleFail
```

**Describe the results you expected:**
Test passed

**Additional information you deem important:**
The scripts based on sonic-mgmt 50a6bc19ac435f8a270d707238d2e1f2a8cc084e

**Comment by msosyak** [comment]
I run this code under pdb and found that exception object raised in https://github.com/Azure/sonic-mgmt/blob/master/tests/common/platform/ssh_utils.py#L20 has type `<class 'tests.common.errors.RunAnsibleModuleFail'>` same time full class name for `RunAnsibleModuleFail` in this file is `<class 'common.errors.RunAnsibleModuleFail'>`. As you see the name is different. 

I guess the reason is in difference between absolute and relative import statements.
https://github.com/Azure/sonic-mgmt/blob/master/tests/common/platform/ssh_utils.py#L3
https://github.com/Azure/sonic-mgmt/blob/master/tests/common/devices.py#L18
Reference: https://www.python.org/dev/peps/pep-0328/

The pdb session logs:
```
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB set_trace (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /home/msosyak/bfn-sonic-mgmt/tests/common/platform/ssh_utils.py(25)prepare_testbed_ssh_keys()
-> pass
(Pdb) l 23
 18  	    ptfhost.shell('rm -f /root/.ssh/id_rsa*')
 19  	    try:
 20  	        ptfhost.shell('stat /root/.ssh/known_hosts')
 21  	    except RunAnsibleModuleFail:
 22  	        pass # files does not exist
 23  	    except Exception as e:          # added for debug
 24  	        import pdb; pdb.set_trace() #
 25  ->	        pass                        #
 26  	    else:
 27  	        ptfhost.shell('ssh-keygen -f /root/.ssh/known_hosts -R ' + dut_ip)
 28  	
(Pdb) type(e)
<class 'tests.common.errors.RunAnsibleModuleFail'>
(Pdb) RunAnsibleModuleFail 
<class 'common.errors.RunAnsibleModuleFail'>
```

**Comment by msosyak** [comment]
Fixed in https://github.com/Azure/sonic-mgmt/pull/1670, but guess there could be other similar issues

**Comment by daall** [comment]
@msosyak can you check if #1671 fixes this problem? @yxieca fyi

**Comment by msosyak** [comment]
Fixed by #1671
